### PR TITLE
[DOCFIX] Update operation/troubleshooting chinese doc

### PR DESCRIPTION
**Pull request by xichen01**
### What changes are proposed in this pull request?

Update operation/troubleshooting Doc

### Why are the changes needed?

The Chinese contributor/Building-Alluxio-From-Source document is not updated with the latest changes, this PR synchronizes these updates
In the updated content, some document links point to English documents, because there is no Chinese version of these documents at present.

### Does this PR introduce any user facing changes?
no

### CODE REVIEW

File: docs/cn/operation/Troubleshooting.md [hunk 1]
```diff
@@ -10,57 +10,251 @@ priority: 8
...
-```console
-$ export ALLUXIO_WORKER_JAVA_OPTS="$ALLUXIO_JAVA_OPTS -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=6606"
-$ export ALLUXIO_MASTER_JAVA_OPTS="$ALLUXIO_JAVA_OPTS -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=6607"
-$ export ALLUXIO_USER_DEBUG_JAVA_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=6609"
+```shell
+# Java 8
+export ALLUXIO_MASTER_ATTACH_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=60001"
+export ALLUXIO_WORKER_ATTACH_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=60002"
+# Java 11
+
```
Comment (dbw9580) [inline docs/cn/operation/Troubleshooting.md:48]: Remove empty line

File: docs/cn/operation/Troubleshooting.md [hunk 2]
```diff
@@ -10,57 +10,251 @@ priority: 8
...
+`suspend = y/n` 会决定JVM进程是否等待直至调试器连接。如果您希望在命令行中进行调试，设置`suspend = y`。否则，设置 `suspend = n` ，这样就可以避免不必要的等待时间。
+
+设置此参数后，可以添加`-debug`标志来启动调试服务器，例如`bin/alluxio fs -debug ls /`。
+
+完成此设置后，可以进一步学习 attach [attach](#To-attach)。
+
+这样启动该节点上的master或者worker后，使用eclipse或intellij IDE等java开发环境，新建java远程调试配置，设置调试主机名和端口号，然后启动调试连接。如果您设置了断点并且到达了断点处，开发环境会进入调试模式，可以读写当前现场的变量、调用栈、线程列表、表达式评估，也可以执行单步进入、单步跳过、恢复执行、挂起等调试控制。掌握这个技术使得日后的定位问题事半功倍，也会对调试过的代码上下文印象深刻。
+
+### To attach
```
Comment (dbw9580) [inline docs/cn/operation/Troubleshooting.md:73]:
```suggestion
### 连接到待调试的进程
```

File: docs/cn/operation/Troubleshooting.md [hunk 3]
```diff
@@ -10,57 +10,251 @@ priority: 8
...
+当 Alluxio 集群未运行时，该命令将无法收集到一些信息。
+这个子命令将运行 `alluxio getConf` 以收集本地配置信息以及 `alluxio getConf --master --source` 命令来收集从 Master 接收到的配置信息。
+它们都会隐藏证书相关字段的配置。不同的是，如果 Alluxio 集群没有启动，后一个命令会失败。
+
+### 收集Alluxio配置文件
+`collectConfig` 将收集 `${alluxio.work.dir}/conf` 目录下的所有配置文件。
+从 Alluxio 2.4 开始 `collectAlluxioInfo` 将会运行 `alluxio getConf` 命令并打印所有配置属性，并且会隐藏证书相关字段,
+而不是拷贝 `alluxio-site.properties` 文件，因为很多用户会将UFS的证书明文信息放在此文件中。
+
+[getConf 命令]({{ '/en/operation/User-CLI.html#getconf' | relativize_url }}) 将收集所有当前节点配置。
```
Comment (dbw9580) [inline docs/cn/operation/Troubleshooting.md:108]:
```suggestion
[getConf 命令]({{ '/en/operation/User-CLI.html' | relativize_url }}#getconf) 将收集所有当前节点配置。
```

File: docs/cn/operation/Troubleshooting.md [hunk 4]
```diff
@@ -126,28 +336,99 @@ Alluxio通过配置`alluxio.security.authentication.type`来提供不同的[用
...
+
+### 问题：我正在运行一个Alluxio shell命令，它卡死了，并且没有任何输出。为什么？
+
+答：大多数Alluxio shell命令都需要连接到Alluxio Master才能正常执行。
+如果命令连接 Master 失败它就会不断重试，看起来就像是 "卡住" 很长时间。
+当然，也有可能是一些命令本身就需要很长时间才能执行完毕，比如在一个速度较慢的UFS上持久化一个大文件。
+如果您想知道在 UFS 中发生了什么，可以检查用户日志（默认存储路径为`${ALLUXIO_HOME}/logs/user_${USER_NAME}.log`）
+或Master日志（默认存储路径为`${ALLUXIO_HOME}/logs/master.log`，位于Master点）。
+
+如果日志中的信息不足以丁文问题，您可以[开启更详细日志]({{ '/en/operation/Basic-Logging.html#enabling-advanced-logging' | relativize_url }})。
```
Comment (dbw9580) [inline docs/cn/operation/Troubleshooting.md:399]:
```suggestion
如果日志中的信息不足以定位问题，您可以[开启更详细日志]({{ '/en/operation/Basic-Logging.html' | relativize_url }}#enabling-advanced-logging)。
```

File: docs/cn/operation/Troubleshooting.md [hunk 5]
```diff
@@ -126,28 +336,99 @@ Alluxio通过配置`alluxio.security.authentication.type`来提供不同的[用
...
+或[Slack频道](https://alluxio.io/slack)，上提问是请附上完整的环境信息，包括
 - Alluxio版本
 - 操作系统版本
 - Java版本
-- 底层文件系统类型和版本
-- 计算框架类型和版本
-- 集群信息, 如节点个数, 每个节点内存, 数据中心内部还是跨数据中心运行
+- UnderFileSystem 类型和版本
+- Computing framework 的类型和版本
+- 集群的信息，例如节点数量，每个节点的内存大小，数据中心内或跨数据中心的内存大小
```
Comment (dbw9580) [inline docs/cn/operation/Troubleshooting.md:431]: These lines do not need changes.

File: docs/cn/operation/Troubleshooting.md [hunk 6]
```diff
@@ -10,58 +10,251 @@ priority: 8
...
-master和worker日志对于理解Alluxio master节点和worker节点的运行过程是非常有帮助的，当Alluxio运行出现问题时，可以查阅日志发现问题产生原因。如果不清楚错误日志信息，可在[邮件列表](https://groups.google.com/forum/#!forum/alluxio-users)查找，错误日志信息有可能之前已经讨论过。
+Master和Worker日志对于理解Alluxio Master节点和Worker节点的运行过程是非常有用的，当Alluxio运行出现问题时，
+可在[Github issue](https://github.com/Alluxio/alluxio/issues)查找，错误日志的相关信息有可能之前已经讨论过。
+您也可以加入我们的 [Slack 频道](https://slackin.alluxio.io/) 并在那里寻求帮助。
+您可以在 [此处]({{ '/en/operation/Basic-Logging.html' | relativize_url }}#server-logs) 找到有关Alluxio日志的更多详细信息。
+
+当Alluxio运行在客户端侧无法连接的服务器上时，客户端侧的日志会很有用。Alluxio客户端通过log4j生成日志消息，因此日志的位置由使用Alluxio的应用程序的log4j配置确定。
+您可以在 [此处]({{ '/en/operation/Basic-Logging.html' | relativize_url }}#application-logs) 找到有关客户端日志的更多详细信息。
+
+`${ALLUXIO_HOME}/logs/user/`是Alluxioshell的日志。每个用户都有单独的日志文件。
```
Comment (dbw9580) [inline docs/cn/operation/Troubleshooting.md:32]:
```suggestion
`${ALLUXIO_HOME}/logs/user/`是Alluxio shell的日志。每个用户都有单独的日志文件。
```

**Review by dbw9580 — APPROVED** [review]
LGTM!

File: docs/cn/operation/Troubleshooting.md (hunk 6 above)
Comment (xichen01) [reply docs/cn/operation/Troubleshooting.md]: done

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**